src/status_im/contexts/chat/home/new_chat/view.cljs

```diff
@@ -99,25 +102,23 @@
      (if (empty? contacts)
        [no-contacts-view {:theme theme}]
        [gesture/section-list
-        {:key-fn                         :title
-         :sticky-section-headers-enabled false
-         :sections                       (rf/sub [:contacts/filtered-active-sections])
-         :render-section-header-fn       contact-list/contacts-section-header
-         :content-container-style        {:padding-bottom 70}
-         :render-fn                      contact-item-render
-         :scroll-enabled                 @scroll-enabled?
-         :on-scroll                      on-scroll}])
-     (when contacts-selected?
-       [quo/button
-        {:type                :primary
-         :accessibility-label :next-button
-         :container-style     style/chat-button
-         :on-press            (fn []
-                                (if one-contact-selected?
-                                  (rf/dispatch [:chat.ui/start-chat public-key])
-                                  (rf/dispatch [:navigate-to :new-group])))}
-        (if one-contact-selected?
-          (i18n/label :t/chat-with {:selected-user primary-name})
-          (i18n/label :t/setup-group-chat))])]))
-
-(def view (quo.theme/with-theme view-internal))
+        {:key-fn                   :title
+         :sections                 (rf/sub [:contacts/filtered-active-sections])
+         :render-section-header-fn contact-list/contacts-section-header
```

Comment:
:render-section-footer-fn

Reply:
I don't understand

Reply:
(defn footer []
[rn/view {:height 8})])

:render-section-footer-fn footer

instead of condition in header